GPy/models/gp_multiout_regression.py

```diff
@@ -13,26 +13,28 @@
 
 class GPMultioutRegression(SparseGP):
     """
-    Gaussian Process model for scalable multioutput regression
-
-    This is a thin wrapper around the models.GP class, with a set of sensible defaults
-
-    :param X_list: list of input observations corresponding to each output
-    :type X_list: list of numpy arrays
-    :param Y_list: list of observed values related to the different noise models
-    :type Y_list: list of numpy arrays
-    :param kernel: a GPy kernel ** Coregionalized, defaults to RBF ** Coregionalized
-    :type kernel: None | GPy.kernel defaults
-    :likelihoods_list: a list of likelihoods, defaults to list of Gaussian likelihoods
-    :type likelihoods_list: None | a list GPy.likelihoods
-    :param name: model name
-    :type name: string
-    :param W_rank: number tuples of the corregionalization parameters 'W' (see coregionalize kernel documentation)
-    :type W_rank: integer
-    :param kernel_name: name of the kernel
-    :type kernel_name: string
+    Gaussian Process model for multi-output regression without missing data
+
+    This is an implementation of Latent Variable Multiple Output Gaussian Processes (LVMOGP) in [Dai et al. 2017].
+
+    Zhenwen Dai, Mauricio A. Álvarez and Neil D. Lawrence. Efficient Modeling of Latent Information in Supervised Learning using Gaussian Processes. In NIPS, 2017.
+
+    :param X: input observations. Numpy.ndarray
```

Comment:
For consistency with the rest of the doccing I've been doing, it would be good to specify the types like;
'''
:type X: np.ndarray (num_data x input_dim)
'''

etc. so that users know what shapes things should be (perhaps obvious for some, but less obvious for things like Z_row and qU_var_c_W_dim).